lightstep/resource_dashboard.go

```diff
@@ -28,6 +28,25 @@ func getUnifiedQuerySchema() map[string]*schema.Schema {
 			Type:     schema.TypeString,
 			Required: true,
 		},
+		"dependency_map_options": {
+			Type:     schema.TypeList,
```

Comment:
Was originally hoping to use [`TypeMap`](https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-types#typemap) here but because it's not possible to declare specific string keys/validate values (afaict) I went with a pattern I found elsewhere in the provider that uses a `TypeList` with one element.